algorand/go-algorand
# Onetechnical/relstable2.1.6

## Summary

## Highlights

A bug in the algorand debian package would result in the genesis.json for TestNet being applied instead of MainNet. Debian users who upgraded to 2.1.5 should verify that `/var/lib/algorand/genesis.json` is pointing to the correct network after upgrading to this version.

## Changes

1. Other
    * Bug Fixes
        * Use DEFAULT_RELEASE_NETWORK genesis file in debian package
        * Attempt to fix bad genesis file on 2.1.5 debian install

## Test Plan

Standard release testing.

clean install 2.1.4, upgrade 2.1.5, upgrade 2.1.6 (scenario #1)
* verify mainnet config (after 2.1.4)
* verify testnet config (after 2.1.5)

clean install 2.1.4, testnet config, upgrade 2.1.5, upgrade 2.1.6 (scenario #2)
* verify mainnet config (after 2.1.4)
* implement testnet config
* verify testnet config (after 2.1.5)
* verify testnet config (after 2.1.6)

clean install 2.1.5, mainnet config, upgrade 2.1.6 (scenario #3)
* verify testnet config (after 2.1.5)
* implement mainnet config
* verify mainnet config (after 2.1.6)

clean install 2.1.5, upgrade 2.1.6 (scenario #4)
* verify testnet config (after 2.1.5)
* verify testnet config (after 2.1.6)

clean install 2.1.6
* verify mainnet config

clean install 2.1.4, upgrade 2.1.6
* verify mainnet config (after 2.1.4)
* verify mainnet config (after 2.1.6)

clean install 2.1.4, testnet config, upgrade 2.1.6
* verify mainnet config (after 2.1.4)
* implement testnet config
* verify testnet config (after 2.1.6)
